setup.sh

```diff
@@ -44,6 +44,9 @@ prometheis_total=1
 additional_tags = {
   "Environment" = "${ENV}"
 }
+ecs_instance_type = "t2.small"
```

Comment:
Maybe we could look into ordering these variables in a nicer way (either by grouping similar together or alphabetical)

Reply:
could order them